storage/replica.go

```diff
@@ -1681,42 +1686,65 @@ func defaultProposeRaftCommandLocked(r *Replica, p *pendingCmd) error {
 				return err
 			}
 
-			return r.withRaftGroupLocked(true, func(raftGroup *raft.RawNode) error {
-				return raftGroup.ProposeConfChange(raftpb.ConfChange{
-					Type:    changeTypeInternalToRaft[crt.ChangeType],
-					NodeID:  uint64(crt.Replica.ReplicaID),
-					Context: encodedCtx,
-				})
+			return r.withRaftGroupLocked(true, func(raftGroup *raft.RawNode) (bool, error) {
+				// We're proposing a command here so there is no need to wake the
+				// leader if we were quiesced.
+				r.unquiesceLocked()
+				return false, /* !unquiesce */
```

Comment:
I don't like that calling `r.unquiesceLocked()` and returning false is not equivalent to returning true. The return value needs to be named and documented as something other than just `unquiesce`.